source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -15,6 +16,7 @@
 #include "source/common/http/message_impl.h"
 
 #include "absl/strings/escaping.h"
+#include "wrappers.h"
```

Comment:
Where is this header file come from?

Reply:
Sorry, my bad. Took it back. Thanks for noticing!